portfolio/src/main/webapp/script.js

```diff
@@ -18,15 +18,18 @@
  */
 function getData() {
   console.log('Fetching data.');
-  const languageCode = document.getElementById('language').value;
   // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/list-comments?language='+languageCode).then(response => response.json())
-  .then((comments) => { 
+  const responsePromise = 
+    fetch('/list-comments?language='+document.getElementById('language').value)
+    .then(response => response.json())
+    .then((comments) => { 
   const commentsListElement = document.getElementById('server-comments-container');
   commentsListElement.innerHTML = '';
   for(let i = 0; i<comments.length; i++){
     commentsListElement.appendChild(
     createListElement(comments[i]));
+    commentsListElement.appendChild(
+    createListElement(comments[i]));
   }
   });
 }
```

Comment:
Clean up this indentation a bit. When "chaining" method calls and breaking to a new line, you should indent the chained calls.
```
function getData() {
  console.log('Fetching data.');
  // The fetch() function returns a Promise because the request is asynchronous.
  const responsePromise = 
    fetch('/list-comments?language=' + document.getElementById('language').value)
      .then(response => response.json())
      .then((comments) => { 
        const commentsListElement = document.getElementById('server-comments-container');
        commentsListElement.innerHTML = '';
        for(let i = 0; i < comments.length; i++){
          commentsListElement.appendChild(
            createListElement(comments[i]));
          commentsListElement.appendChild(
            createListElement(comments[i]));
        }
  });
}
```
If nothing else, it helps readers easily parse how deep a given line of code is.